src/test/java/pl/touk/sputnik/HttpConnectorEnv.java

```diff
@@ -9,13 +10,17 @@
 
 public class HttpConnectorEnv {
 
-    protected void stubGet(UrlMatchingStrategy url, String responseFile) throws Exception {
+    protected void stubGet(UrlMatchingStrategy url, ResponseDefinitionBuilder responseDefinitionBuilder) {
         stubFor(get(url)
                 .withHeader("Authorization", equalTo("Basic dXNlcjpwYXNz"))
-                .willReturn(aResponse()
+                .willReturn(responseDefinitionBuilder));
+    }
```

Comment:
[FindBugs] WARNING: UI: Usage of GetResource in pl.touk.sputnik.HttpConnectorEnv.stubGet(UrlMatchingStrategy, String) may be unsafe if class is extended